inst/rmd/h/default.html

```diff
@@ -585,7 +650,9 @@ <h4 class="date">$date$</h4>
 
 $$(document).ready(function () {
   $$('.tabset-dropdown > .nav-tabs > li').click(function () {
-    $$(this).parent().toggleClass('nav-tabs-open')
+    $$(this).parent().toggleClass('nav-tabs-open');
+    $$(this).siblings().removeClass("active");
+    $$(this).addClass("active");
```

Comment:
Consider making the class name `active` something with less overloaded meanings, if we only care about making the arrow appear.

Reply:
Done in https://github.com/rstudio/rmarkdown/pull/1706/commits/989bc53cea0902a8faa9473eca216dec2a0b9ddf